main.tf

```diff
@@ -44,7 +44,7 @@ module "logs_label" {
 }
 
 locals {
-  region     = coalesce(var.region, data.aws_region.current.name)
+  region     = coalesce(var.region, data.aws_region.current.id)
```

Comment:
```suggestion
  region     = coalesce(var.region, data.aws_region.region)
```
@westonplatter I don't see the `id` argument in the docs:
<img width="674" alt="Screenshot 2025-06-29 at 13 45 19" src="https://github.com/user-attachments/assets/cb1325dc-41e7-4847-85a2-0d7c4688bd56" />

Reply:
Thanks for double checking the docs. Yeah, `id` isn't a valid reference. Based on the note for `name`, I agree with you suggestion of using `region`. I think though we need to keep the `current` reference to the data element, so,

`data.aws_region.current.region`

Reply:
Sorry, we should obviously keep the `current` 😃